Docker/Dockerfile.environment

```diff
@@ -1,20 +1,23 @@
 FROM jamesmcclain/crossbuild:0
 LABEL maintainer="James McClain <james.mcclain@gmail.com>"
 
-RUN apt-get update -q && \
+RUN apt-get update -y && \
     apt-get install build-essential pkg-config openjdk-8-jdk -y -q && \
     apt-get autoremove && \
     apt-get autoclean && \
     apt-get clean
 
-# Build GDAL 2.4.3
+# Install SQLite
+RUN apt-get install -y sqlite3 libsqlite3-dev
+
+# Build GDAL 3.1.2
 RUN cd /usr/local/src && \
-    wget -k 'https://download.osgeo.org/gdal/2.4.3/gdal-2.4.3.tar.gz' && \
-    wget -k 'https://download.osgeo.org/proj/proj-4.9.3.tar.gz' && \
-    tar axvf gdal-2.4.3.tar.gz && tar axvf proj-4.9.3.tar.gz && \
-    cd proj-4.9.3 && ./configure --prefix=/usr/local && nice -n 19 make -j33 && make install && \
-    cd ../gdal-2.4.3 && ./configure --prefix=/usr/local && nice -n 19 make -j33 && make install && \
-    cd .. && rm -r proj-4.9.3/ gdal-2.4.3/ proj-4.9.3.tar.gz gdal-2.4.3.tar.gz
+    wget -k 'https://download.osgeo.org/gdal/3.1.2/gdal-3.1.2.tar.gz' && \
+    wget -k 'https://download.osgeo.org/proj/proj-6.0.0.tar.gz' && \
```

Comment:
Hm, this is not related to any bugs, but probably I'll bump proj version up to 7.1.0 to keep deps up to date. However, it would require to add curl and libtiff deps into the build.